pkg/apis/servicecatalog/validation/instance.go

```diff
@@ -140,13 +140,13 @@ func validateServiceInstanceStatus(status *sc.ServiceInstanceStatus, fldPath *fi
 	}
 
 	switch status.CurrentOperation {
-	case sc.ServiceInstanceOperationProvision, sc.ServiceInstanceOperationUpdate:
+	case sc.ServiceInstanceOperationProvision, sc.ServiceInstanceOperationUpdate, sc.ServiceInstanceOperationDeprovision:
```

Comment:
Since this is all of the operations, should we just check `status.CurrentOperation != ""`?